datasources/jito-shredstream-grpc-datasource/src/lib.rs

```diff
@@ -105,67 +131,113 @@ impl Datasource for JitoShredstreamGrpcClient {
                         let total_entries = entries.len();
                         let mut duplicate_entries = 0;
 
-                        for entry in entries {
+                        'next_entry: for entry in entries {
                             if dedup_cache.contains(&entry.hash) {
                                 duplicate_entries += 1;
                                 continue;
                             }
                             let _ = dedup_cache.put(entry.hash, ());
 
                             for transaction in entry.transactions {
+                                let accounts = transaction.message.static_account_keys();
+                                let is_vote = accounts.len() == 3 && accounts[2].eq(&VOTE_ID);
```

Comment:
or

```suggestion
                                let is_vote = accounts.len() == 3 && solana_sdk::vote::program::check_id(&accounts[2]);
```

if you prefer